src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -198,15 +197,7 @@ export class TxSender {
     }
 
     const tx = isExecution ? getExecutionTransaction(txArgs) : getApprovalTransaction(this.safeInstance, safeTxHash)
-
-    const sendParams: PayableTx = {
-      from,
-      value: 0,
-      gas: txProps.ethParameters?.ethGasLimit,
-      [getGasParam()]: txProps.ethParameters?.ethGasPriceInGWei,
-      nonce: txProps.ethParameters?.ethNonce,
-    }
-
+    const sendParams = createSendParams(from, txProps.ethParameters || {})
```

Comment:
The fallback to an empty object could cause issues in `createSendParams` since we're trying to access properties that might not be there.

Reply:
I didn't touch that part, looks like some github glitch when rebasing to dev.